schema.ts

```diff
@@ -839,9 +839,18 @@ export interface Identity extends IdentityDefinition {
   packId: number;
 }
 
+export interface ObjectProperty {
+  label: string;
+  value: string;
+}
+
+export type PropertyType<K extends string = string> = K | string | ObjectProperty;
```

Comment:
Nit, PropertyIdentifier may be a little clearer. The properties themselves are the contents of the `properties` dict in the schema def, and those property defs have a `type` field; this type has nothing to do with those values and that field, so we probably want to distinguish the naming.

Similar, ObjectProperty might be better as `PropertyIdentifierSpec` or something.

Reply:
mmm yeah good call, updated